### PR TITLE
docs: Update Alyssa on GDE list

### PR DESCRIPTION
**Pull request by alyssamichelle**
# docs: Update Alyssa on GDE list
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [x] Other... Please describe: Update to aio/marketing files for GDE list

## What is the current behavior?
Issue Number: N/A

## What is the new behavior?
New updated photo and bio for Alyssa on GDE list.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by alyssamichelle** [comment]
@splaktar Can I get a review? :D

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -280,10 +280,10 @@
   },
   "alyssa": {
     "name": "Alyssa Nicoll",
-    "picture": "mickey_mouse.jpg",
+    "picture": "alyssa.jpg",
     "twitter": "alyssanicoll",
     "website": "alyssa.io",
-    "bio": "I am an energetic, über passionate GDE and Web Dev. I have some Front-End and Angular courses on Egghead.io and Code School. I love to learn new things and share them with others. I Scuba Dive and have a toothless dog named 'Gummy'. My DM is always open, come talk sometime.",
+    "bio": "Alyssa is an Angular Developer Advocate for KUI and a GDE. Her two degrees (Web Design & Development and Psychology) feed her speaking career. She has spoken at over 35 conferences Internationally. She is a weekly panelist on Adventures in Angular and Angular Air, which have a combined following of over 16,000 listeners. She enjoys gaming, scuba diving, and has a newborn that fondly goes by 'Mr. Milks'.",
```
Comment (Splaktar) [inline aio/content/marketing/contributors.json]:
```suggestion
    "bio": "Alyssa is an Angular Developer Advocate for KUI and a GDE. Her two degrees (Web Design & Development and Psychology) feed her speaking career. She has spoken at over 35 conferences internationally. She is a weekly panelist on Adventures in Angular and Angular Air, which have a combined following of over 16,000 listeners. She enjoys gaming, scuba diving, and has a newborn that fondly goes by 'Mr. Milks'.",
```

**Review by Splaktar — CHANGES_REQUESTED** [review]
As mentioned by the lint task that failed in CircleCI (`The commit message header does not match the format of '<type>(<scope>): <subject>'`), you'll need to add `docs: ` in front of your commit message (and you should add that to the PR title too).

Please resize your photo down to something that is about 20 KBs. The height and width should be around 170 x 170.

**Comment by Splaktar** [comment]
The only thing left to do is to squash your commits, [instructions](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-squash) (from @kapunahelewong). Oh and I would also update the commit message to match the PR title during your squashing.

**Comment by kapunahelewong** [comment]
Hi @alyssamichelle! Thanks for this update. To get your picture down to about 20k, you could try lowering the jpg quality if you haven't already. That should help while keeping it around 170x170.

For the commit messages, you'll want to squash and then do a `git commit --amend` and change the commit message to something like:

```
docs: update Alyssa on GDE list
```
That commit message should pass.

You may also like [my own notes on squashing and amending](https://gist.github.com/kapunahelewong/567c30f7b353fd1c39bbf6d8343e2dca#how-to-squash). They go step-by-step and are specific to this repo.

If you have any questions just let me know and when you're all ready, please tag me. Congratulations on Mr. Milks! 🌟🍼

**Comment by alyssamichelle** [comment]
Thanks @Splaktar and @kapunahelewong !! 

Killer article @kapunahelewong btw 👏

**Comment by kapunahelewong** [comment]
ohhh thank you, @alyssamichelle. No blush emoji red enough 😊

**Review by Splaktar — APPROVED** [review]
LGTM

**Review by StephenFluin — APPROVED** [review]

**Pull request closed without merging**